mesh/accounts/models.py

```diff
@@ -7,15 +7,8 @@ class Account(models.Model):
 
     #please set editable to False after debugging process
     encryptedPass = models.BinaryField(max_length=64,editable = True)
-    salt = models.BinaryField(max_length = 36,editable = True) 
-    ###
+    salt = models.BinaryField(max_length = 36,editable = True)
     phoneNum = models.CharField(max_length=15)
-    DISPLAY_THEMES = [
-        ('L', 'Light'),
-        ('D', 'Dark'),
-    ]
-    displayTheme = models.CharField(max_length=1, choices=DISPLAY_THEMES, default='L')
-    enabled2Factor = models.BooleanField(default=False)
```

Comment:
Just to confirm, were these properties moved to some other table?

Reply:
yes it has moved to the accountSettings model